rmw_connext_cpp/CMakeLists.txt

```diff
@@ -19,12 +19,13 @@ endif()
 find_package(rmw REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
 find_package(rosidl_typesupport_connext_cpp REQUIRED)
+find_package(rmw_connext_shared_cpp)
 
 ament_export_definitions(${Connext_DEFINITIONS})
 ament_export_dependencies(rmw rosidl_generator_cpp rosidl_typesupport_connext_cpp)
 ament_export_include_directories("${Connext_INCLUDE_DIRS}")
 
-link_directories(${Connext_LIBRARY_DIRS})
+link_directories(${rmw_connext_shared_cpp_LIBRARY_DIRS})
```

Comment:
For an ament package we don't need `LIBRARY_DIRS` since we always link against absolute path. Should this link against the library of `rmw_connext_shared_cpp` though?